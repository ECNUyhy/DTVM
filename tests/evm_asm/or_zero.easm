// X | 0 = X
PUSH1 0x00
PUSH1 0xAB
OR
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
