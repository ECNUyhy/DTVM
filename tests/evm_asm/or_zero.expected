status: success
error_code: 0
stack: []
memory: '00000000000000000000000000000000000000000000000000000000000000AB'
storage: {}
transient_storage: {}
return: '00000000000000000000000000000000000000000000000000000000000000AB'
events: []
